src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
Consider using a shortcut for method definitions in the object. Instead of `function makeCalculator() {`, you can simply use `const makeCalculator = () => {`.